homeassistant/util/package.py

```diff
@@ -29,7 +29,11 @@ def is_virtual_env() -> bool:
 @cache
 def is_docker_env() -> bool:
     """Return True if we run in a docker env."""
```

Comment:
Consider renaming this to:

```
Return True if we run in a container env.
```

The container runtime shouldn't matter

Reply:
Agreed, thank you